api/contact.py

```diff
@@ -33,19 +40,61 @@ def contact():
 
             logging.debug(f"New submission: {new_submission}")
 
-            supabase.table('submissions').insert(new_submission).execute()
+            # Insertar en Supabase (código existente)
+            supabase.table('contact_submissions').insert(new_submission).execute()
+            logging.info("Submission successfully inserted into Supabase.")
+
+            # Lógica para enviar el email de notificación con Resend
+            if RESEND_API_KEY and YOUR_EMAIL_1 and YOUR_EMAIL_2 and RESEND_DEMO_SENDER_EMAIL:
```

Comment:
What are these environment variables for? Did you mean to check if there is an email in `new_submission` dict?

```suggestion
            if RESEND_API_KEY and new_submission['email'] and new_submission['name']:
```

Reply:
The environment variables serve to store and maintain privacy of the emails that are going to receive the notification from Resend API. The funtionality of Resend is to send the contact form submit information in a 'Email format' to some emails that are specify in the environment variables.

Reply:
Those emails (example: YOUR_EMAIL_1 and YOUR_EMAIL_2) belongs to us the owners of the website to get an email notification when a user ask some questions on the contact form.

Reply:
Oh! okok. Got it. 
In that case, maybe a better environment variable name would be `BUSINESS_EMAIL_1` instead of `YOUR_EMAIL_1`. And same thing for the other one

Reply:
Ok nice, I'll do that!